### DISCUSSION TITLE
Looking for migration advice

### DISCUSSION THREAD

**Discussion by dprothero** [twilio-labs/paste#2091] [Q&A] [answered]
The [Internal Product Docs](https://internal-product-docs.twilio.com) is currently built using the [Match](https://match.twilio.design/) design system (the design system for Brand). I am looking at migrating to using Paste and looking for some advice.

1. Solid Twilio branding is important to me. Can I override the fonts to use Brand standard fonts Whitney and FiraMono? I've got the [fonts working now](https://code.hq.twilio.com/twilio/internal-product-docs/blob/main/src/css/custom.css#L1), so the Q is just mainly if I can override the fonts that Paste uses.

2. It looks like Paste has a .css file with variables for the tokens that I can import. That's basically the bulk of what I need to make use of, as the site is built on Docusaurus, and I have to map their CSS variables to the design system's as [shown here](https://code.hq.twilio.com/twilio/internal-product-docs/blob/main/src/css/custom.css#L333). Is this similarly straight-forward with Paste, or will I run into gotchas?

3. The Match components I am using are: [Anchor](https://match.twilio.design/components/anchor/), [Card](https://match.twilio.design/components/card/), [Heading](https://match.twilio.design/components/heading/), [Paragraph](https://match.twilio.design/components/paragraph/), [Grid, Cell](https://match.twilio.design/components/grid/), [List, and ListItem](https://match.twilio.design/components/list/). Any concerns with moving to Paste components here?

Thanks!

**Comment by loreina** [comment, marked as answer]
Hi @dprothero!

1. You can [create a custom theme](https://paste.twilio.design/customization/creating-a-custom-theme) to override the fonts and import the fonts you want to use. You’ll have to import Whitney but Paste’s default theme uses Fira Mono for monospace.
2. Yeah, your assumption here is correct. It should be as straight forward as it seems.
3. We have most of those components available in Paste so no concerns come to mind. We don’t have a Cell component, but you can find guidance on using Grid on our doc site: https://paste.twilio.design/layout/grid

**Reply by dprothero** [reply]
Awesome, thank you!
